app/views/registrations/school_profile/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% content_for :before_content, govuk_back_link(
+      text: "Back",
+      href: registrations_question_two_path)
```

Comment:
Should the back links all go back to the same page or should they go back to the previous step?

Reply:
To the previous step